src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseContentFiles.cs

```diff
@@ -26,7 +25,7 @@ public GivenThatWeWantToUseContentFiles(ITestOutputHelper log) : base(log)
         [Fact]
         public void It_handles_content_files_correctly()
```

Comment:
Rather than updating the existing test to cover more scenarios, I'd prefer adding a test that specifically covers incremental build.

Reply:
I consider it, but this saves an extra build for an already long run time for a unit test.